src/security/cors.rs

```diff
@@ -148,18 +136,18 @@ impl CorsMiddleware {
 impl<State: Send + Sync + 'static> Middleware<State> for CorsMiddleware {
     fn handle<'a>(&'a self, req: Request<State>, next: Next<'a, State>) -> BoxFuture<'a, Result> {
         Box::pin(async move {
-            let origins = req.header(&headers::ORIGIN).cloned().unwrap_or_default();
-
             // TODO: how should multiple origin values be handled?
-            let origin = match origins.first() {
-                Some(origin) => origin,
+            let origins = match req.header(headers::ORIGIN).cloned() {
```

Comment:
Can be simplified?
```suggestion
            let origin = match req.header(headers::ORIGIN).cloned() {
                Some(origins) => origin.last(),
```

Reply:
What do you have in mind?

Reply:
See the suggestion code - return `origin` from the `match` by already getting `.last()` there.

Reply:
Ahh, I just tried it and remembered why it's like this — we use `origins` later https://github.com/http-rs/tide/blob/20c1896d5352b65e6eab0f177c8c7009048021cb/src/security/cors.rs#L156

Reply:
Ah, didn't see that.